caravel/viz.py

```diff
@@ -1316,6 +1316,7 @@ class DistributionBarViz(DistributionPieViz):
             ('y_axis_format', 'bottom_margin'),
             ('x_axis_label', 'y_axis_label'),
             ('reduce_x_ticks', 'contribution'),
+            ('order_bars'),
```

Comment:
`('order_bars', None)` will render using half the column so that the checkbox is not too far away from the label.